### PR TITLE
Cura 9365 fix building main

### PR DESCRIPTION
**Pull request by jellespijker**
Doc to come

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +1,268 @@
...
+> ```shell
+> conan profile new default --detect  # This should create a default profile with the system detected standard compiler
+> conan config install https://github.com/ultimaker/conan-config.git  # This will install our profiles, settinsg and remotes
+> ```
+
+As mentioned in the notes above we use our own JFrog Artifactory server, so make sure you add this server tou your Conan
+remotes. **This step is not necessary if you use our supplied Conan configuration.**
+
+```shell
+conan remote add ultimaker https://peer23peer.jfrog.io/artifactory/api/conan/cura-conan True
```
Comment (Joeydelarago) [inline README.md:64]: I think this should be updated to our pulic ultimaker artifactory

File: docs/development/zlib.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+#include <zconf.h>
+#include <zlib.h>
+```
+
+<br>
+
+
+---
+---
+Conan **1.47.0**. JFrog LTD. [https://conan.io](https://conan.io). Autogenerated 2022-04-26 15:36:12.
```
Comment (Joeydelarago) [inline docs/development/zlib.md:243]: Is this file needed? I see it is autogenerated.

Comment (jellespijker) [reply docs/development/zlib.md:243]: wrong PR @Joeydelarago you should review the 5.1
